inc/toolbox.class.php

```diff
@@ -955,11 +955,33 @@ static function commonCheckForUseGLPI() {
          'gd'       => [
             'required'  => true,
          ],
+         'simplexml' => [
+            'required'  => true,
+         ],
+         'xml'        => [
+            'required'  => true,
+            'function'  => 'utf8_decode'
+         ],
+         //to sync/connect from LDAP
          'ldap'       => [
             'required'  => false,
          ],
+         //for mail collector
          'imap'       => [
             'required'  => false,
+         ],
+         //to enhance perfs
+         'Zend OPcache' => [
+            'required'  => false
+         ],
+         //to enhance perfs
+         (PHP_MAJOR_VERSION < 7 ? 'APCu' : 'apcu-bc') => [
+            'required'  => false,
+            'function'  => 'apc_fetch'
```

Comment:
Yes, I plan to rerun compatinfo against master anyways, things may have changed :)